crates/completest-pty/src/lib.rs

```diff
@@ -217,8 +217,11 @@ PS1='% '
     pub fn complete(&mut self, input: &str, term: &Term) -> std::io::Result<String> {
         let mut command = Command::new("bash");
         let inputrc_path = self.home.join(".inputrc");
+        // If TERM env var is unset, the bash prompt would be duplicated.
+        let env_term = std::env::var_os("TERM").unwrap_or_else(|| "dumb".into());
```

Comment:
As the host terminal is not being exposed to bash, we should not inherit from the environment set for the host terminal

Reply:
So do you suggest to just set TERM to a fixed value? In this case, "TERM=dumb".